sprocs/instance_questions_calculate_stats.sql

```diff
@@ -0,0 +1,103 @@
+CREATE OR REPLACE FUNCTION
+    instance_questions_calculate_stats(
+        instance_question_id_var bigint
+    ) RETURNS VOID
+AS $$
+BEGIN
+    WITH more_info AS (
+        SELECT
+            aq.assessment_id,
+            aq.question_id,
+            a.course_instance_id,
+            aq.id AS assessment_question_id
+        FROM
+            instance_questions AS iq
+            JOIN assessments AS a ON (a.id = iq.assessment_id)
```

Comment:
This fails because there is no column `instance_questions.assessment_id`.

Reply:
Not sure why I didn't catch this... the migration version of the code doesn't seem to have this bug.

Reply:
Fixed in the next commit.